bindings/python/iota_sdk/types/transaction_options.py

```diff
@@ -57,7 +38,6 @@ def to_dict(self) -> dict:
 
         return {
             'strategy': self.name,
-            'value': self.value[0]
```

Comment:
Why is value not part of the serialisation?

Reply:
Oh the diff mislead me, this is RemainderValueStrategy

Reply:
Why is it done like that btw? Why an enum just for ReuseAddress?

Reply:
We had a second option to generate a new remainder address and didn't you also say that we might add a new remainder option in the future?
Currently I could think of a remainder option that will use the account address instead of the ed25519 one

Reply:
I'm confused, that's not really what I'm asking. We have `RemainderValueStrategy` which is an enum that has one `ReuseAddress` variant and then we have `RemainderValueStrategyCustomAddress` which is not part of the enum?

Reply:
Ah it's not part of the enum because you can't have class things as enum variants

Reply:
bullshit